src/ds5/ds5-device.cpp

```diff
@@ -857,6 +858,15 @@ namespace librealsense
                 []() {return std::make_shared<y12i_to_y16y16>(); }
             );
 
+            if (_pid == RS_D585_PID || _pid == RS_S585_PID)
+            {
+                depth_sensor.register_processing_block(
+                    { RS2_FORMAT_Y16I },
+                    { {RS2_FORMAT_Y16, RS2_STREAM_INFRARED, 1}, {RS2_FORMAT_Y16, RS2_STREAM_INFRARED, 2} },
```

Comment:
1 function above we register Y12i --> Y16 for all.

On D/S585 the Y16I should replace it right?

Reply:
?